zork++/src/lib/cli/mod.rs

```diff
@@ -30,49 +29,53 @@ use clap::{Parser, Subcommand, ValueEnum};
 )]
 pub struct CliArgs {
     #[command(subcommand)]
-    pub command: Option<Command>,
+    pub command: Command,
```

Comment:
Another person took care about the parser, but as far as I remember, without the optional doesn't mean that there's should always a command present?

I mean, for now, build the project usually was by just typing in the command line `zork++`, to invoke the executable.

But another interesting point would be align this behaviour with cargo, for example, explicitly type `zork++ build` for compile the project, and `zork++ run` for build and run the executable generated.

I think that this could make a lot of sense, and get are able to get rid out of the `autorun` option in the config file. 

What do you think about that?

Reply:
That was exactly my intention. The behavior of `cargo` is to print the usage page if no command is specified.
This also aligns with other Unix tools. Many tools don't do anything if you don't provide arguments, including other build tools such as `cargo`, `gradle` or even `cmake`.

The only comparable tool that does otherwise is plain `make`.

`cargo` is popular and well received as far as I can tell. That would be the route I personally would take.

Reply:
It seems the more reasonable way-to-go for me as well